### ISSUE TITLE
Crash report 7.4: NullPointerException in NetworkImageView.loadImageIfNecessary

### ISSUE THREAD

**Issue by theck13** [wordpress-mobile/WordPress-Android#6002]
The crash is exclusive to Mito A260 devices running Android 4.

```
Fatal Exception: java.lang.NullPointerException
       at com.android.volley.toolbox.NetworkImageView.loadImageIfNecessary(NetworkImageView.java:160)
       at com.android.volley.toolbox.NetworkImageView.onLayout(NetworkImageView.java:209)
       at android.view.View.layout(View.java:15140)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
       at android.widget.LinearLayout.layoutHorizontal(LinearLayout.java:1877)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1653)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1742)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1651)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
       at android.widget.ScrollView.onLayout(ScrollView.java:1479)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.support.v7.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:437)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1742)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1651)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
       at android.view.View.layout(View.java:15140)
       at android.view.ViewGroup.layout(ViewGroup.java:4867)
       at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2480)
       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2175)
       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1249)
       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6420)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:788)
       at android.view.Choreographer.doCallbacks(Choreographer.java:591)
       at android.view.Choreographer.doFrame(Choreographer.java:560)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:774)
       at android.os.Handler.handleCallback(Handler.java:808)
       at android.os.Handler.dispatchMessage(Handler.java:103)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:5299)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:825)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:641)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by daniloercoli** [comment]
Not sure what we can do here. The only option I see to fix this issue here is to replace the few instances of `com.android.volley.toolbox.NetworkImageView` with our implementation of it `WPNetworkImageView`. Maybe we've already added additional checks that prevent this crash. Or we can just add them to our code, once we know where exactly the code is crashing

From my finding/betting, I think the `response` is null here:
https://github.com/google/volley/blob/1.0.0/src/main/java/com/android/volley/toolbox/NetworkImageView.java#L160 or somewhere in the listener. We may want to provide a patch for this issue into the original prj too, or resubmit this issue on their Github site.


cc @nbradbury

**Comment by nbradbury** [comment]
Looks like we have the same problem [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java#L275) in `WPNetworkImageView`.

**Comment by nbradbury** [comment]
Closed via #6033
